+bids/query.m

```diff
@@ -257,7 +257,7 @@
 
             case 'dependencies'
               if isfield(d(k), 'dependencies')
-                result{end + 1} = d(k).dependencies;
+                result{end + 1, 1} = d(k).dependencies;
```

Comment:
In general query returns a array of strings, except for dependencies, in which case it will return an array of structure.

Reply:
Yeah metadata is also handled that way.

If you ask info about a single file then you get a structure.

If you ask for metada or dependencies about several files, you can get structures that have different shapes so you can't concatenate them in something prettier and easier to handle. :sob: